docs/pages/eas/environment-variables/index.mdx

```diff
@@ -0,0 +1,142 @@
+
+Since **Expo Application Services (EAS)** such as EAS Build and EAS Workflows jobs run on a remote server, **.env** files from your local machine are not available to these jobs. For instance, if **.env** files are excluded from project's version control, that is they are listed in the **.gitignore** file or not committed.
+
+Additionally, you may want to use environment variables outside of your project's code to customize your app binary at build time. For example, you want to set bundle identifier or a private key for an error reporting service. To accommodate these needs, EAS provides a separate mechanism for environment variables. This mechanism is focused on storing and managing environment variables oon EAS servers and synchronizing them for local development.
+
+## Quick start
+
+<Step label="1">
+
+To create a new environment variable, use the EAS CLI and run the following command inside your project directory. The following command creates a new environment variable with the name `EXPO_PUBLIC_API_URL` and the value `https://api.example.com` for the `production` environment.
+
+<Terminal
+  cmd={[
+    '$ eas env:create --name EXPO_PUBLIC_API_URL --value https://api.example.com --environment production --visibility plainText',
+  ]}
+/>
+
+</Step>
+
+<Step label="2">
+
+Verify if the environment variable is created successfully by confirming it appears with the **production** badge in the [Environment variables](https://expo.dev/accounts/[account]/projects/[project]/environment-variables) in your project settings. You canm
+
+</Step>
+
+<Step label="3">
+
+To use the environment variable in EAS Build, add `environment` field to the `prodcuction` build profile:
+
+```json eas.json
+{
+  "build": {
+    "production": {
+      "environment": "production"
+    }
+  }
+}
+```
+
+Any environment variable that is created for the `production` build profile will be available during the build process.
+
+</Step>
+
+<Step label="4">
+
+To use the environment variable with EAS Update, run the `eas update` command with the `--environment` flag specified:
+
+<Terminal cmd={['$ eas update --environment production']} />
+
+The `--environment` flag is used to specify the environment to use for the update job. Only the environment variables from the specified environment will be used during the update process.
+
+</Step>
+
+<Step label="5">
+
+To use the environment variable with EAS Hosting, run the `eas deploy` command with the `--environment` flag specified:
+
+<Terminal cmd={['$ eas deploy --environment production']} />
+
+The `--environment` flag is used to specify the environment to use for the deploy job. Only the environment variables from the specified environment will be used during the deploy process.
+
+</Step>
+
+## Why use EAS environment variables
```

Comment:
Could this section can be near the top? I added a suggestion that had bullet points which overlap with this, this could essentially replace my bullet point suggestion.